app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/HelloWorldPage.scala.html

```diff
@@ -22,7 +22,7 @@
 @()(implicit messages: Messages, appConfig: AppConfig)
 
 @layout(pageTitle = Some("cds-reimbursement-claim-frontend")) {
```

Comment:
We should remove this HelloWorldPage and the respective controller and endpoint now that we have a StartPage

Reply:
Please give it 2 more weeks, the QA's haven't even seen the page on Dev or QA.
At the moment this is the only page they can come to see if the service works at all.